update: Compose Multiplatform 1.6.10 release (#135)
JetBrains/kotlin-multiplatform-dev-docs: topics/compose/compose-images-resources.md

```

For example:

```kotlin
coroutineScope.launch {
    val appName = getStringArray(Res.string.str_arr)
}
```

</tab>
</tabs>

#### Plurals

When your UI displays quantities of something, you might want to support grammatical agreement for different numbers
of the same thing (one _book_, many _books_, and so on) without creating programmatically unrelated strings.

The concept and base implementation in Compose Multiplatform are the same as for quantity strings
on Android.
See [the Android documentation](https://developer.android.com/guide/topics/resources/string-resource#Plurals) for more
about best practices and nuances of using plurals in your project.

* The supported variants are `zero`, `one`, `two`, `few`, `many`, and `other`. Note that not all variants are even 
  considered for every language: for example, `zero` is ignored for English because it is the same as any other plural
  except 1. Rely on a language specialist to know what distinctions the language actually insists upon.
* It's often possible to avoid quantity strings by using quantity-neutral formulations such as "Books: 1".
  If this doesn't worsen the user experience, 

> To define a plural, add a `<plurals>` element to any .xml file in your `composeResources/values` directory.

Reviewer: ```suggestion
To define a plural, add a `<plurals>` element to any `.xml` file in your `composeResources/values` directory.
```

Reviewer: Nitpicking

* [Kamel](https://github.com/Kamel-Media/Kamel)
* [Ktor client](https://ktor.io/)

### Accessing resources from external libraries

> If you would like to make Compose Multiplatform resources available to other libraries used in your project, you

Reviewer: ```suggestion
If you want to make Compose Multiplatform resources available to other libraries used in your project, you
```

Reviewer: Nitpicking

can call the `Res.getUri()` function with the general path of the resource:

```kotlin
val uri = Res.getUri("files/my_video.mp4")
```

The `uri` variable will contain the precise platform-specific path to the file that the path points to.
External libraries can use that path to access the file in a manner that suits them.

## Publication

Starting with Compose Multiplatform 1.6.10-beta01, all necessary resources are included in the publication
maven artifacts.

> To enable this functionality, your project needs to use Kotlin 2.0.0-Beta5 or newer and Gradle 7.6 or newer.

Reviewer: ```suggestion
To enable this functionality, your project needs to use Kotlin 2.0.0 or newer and Gradle 7.6 or newer.
```

Reviewer: Is that OK?
